src/components/App.js

```diff
@@ -146,26 +116,12 @@ class App extends Component {
   renderedTotalInventory() {
     if (this.state.data[0]['Inventory Amount'] > 15) {
```

Comment:
What is the significance of 15? You should use a variable name to make this apparent other developers.